docs/wip/build-pipeline.md

```diff
@@ -1,7 +1,95 @@
+
+Or directly on this url: `https://preview-<pull request number>--<your store name>.vtex.app/`.  
+For example: `https://preview-417--storecomponents.vtex.app/`.
+
+
+## SonarQube
+
+[SonarQube](sonarqube.md) is used to measure and analyze code quality using static analysis.
+
+Every time that the CI flow is triggered, a static analysis of the code is made taking into account the latest changes of the pull request. If any issues are found by sonarqube, they will be marked as annotations on the pull request like so:
+
+![Store Framework Jamstack CI sonarqube annotations](./../images/sonarqube_annotation.png)
+
+#### Possible states
+- `queued`: soon this step will be executed.
+- `in_progress`: this step is being executed.
+- `completed/success`: Sonarqube didn't found any critical issues.
+- `completed/failure`: There is at least one critical issue found by sonarqube.
+
+More information can be found [here](./sonarqube.md)
 
 ## Lighthouse CI
 
-## Explain each CI that we do
+Lighthouse is a tool that is used to monitor the performance of a website, and it is executed each time a new build is completed, using the `Deploy Preview` as the source of analysis. These results are aggregated and allow the developer to compare the performance of whats being developed with what is in production.  
+
+You can access the execution logs by clicking on `details` at the `Lighthouse` check.  
+![Store Framework Jamstack CI lhci check completed](./../images/lhci_check_completed.png)
+
+#### Possible states
+- `queued`: soon this step will be executed.
+- `in_progress`: this step is being executed.
+- `completed/success`: Lighthouse CI is completed successfully.
+- `completed/failure`: Something went wrong.
+- `completed/canceled`: The build was not successful, therefore the Lighthouse CI can't be executed.
+
+
+## Integration Tests
+
+Store Framework Jamstack CI flow support integration tests implemented with Cypress.
+Each time a build is completed, the integrations tests are executed using as URL the `Deploy Preview` generated with the latest changes on the current pull request.
+
+The results of the tests can be seen directly on GitHub, by clicking on `details` at the `Integration Tests` check.
+
+![Store Framework Jamstack CI integration tests check completed](./../images/integration_tests_check_completed.png)
+
+![Store Framework Jamstack CI integration tests results](./../images/integration_tests_results.png)
+
+More information can be found [here](./e2e-testing.md)
 
-- build, integration test e2e, lighthouse, sonarqube
+#### Possible states
+- `queued`: soon this step will be executed.
+- `in_progress`: this step is being executed.
+- `completed/success`: All tests passed successfully.
+- `completed/failure`: At least one test failed.
+- `completed/canceled`: The build was not successful, therefore the Integrations Tests can't be executed.
+- `completed/neutral`: There are no integration tests implemented.
```

Comment:
```suggestion
```

Reply:
It is important to explicit the possible states of LHCI and Integration tests steps as they are influenced by the result of the Build Step.